app/components/validated_field_component.html.erb

```diff
@@ -0,0 +1,28 @@
+<lg-validated-field>
+  <%= content_tag(
```

Comment:
Note for future consideration: Not sure if we should want to be populating strings like this, vs. leveraging our existing Webpack string extraction. On one hand, it improves portability of the component to think about how it might be used outside our toolchain, e.g. for design system consideration. On the other hand, it works differently, and loses out on ease-of-use of the Webpack approach.

Also worth noting that it probably won't work out of the box as-is unfortunately, because I've found the Webpack plugin is quite naive in how it identifies strings, and isn't very flexible to how Webpack would internally rewrite imports to `import { t } from '@18f/identity-i18n'`. Last time I looked, improving this should likely happen with/after the Webpacker 6 (Webpack 5) upgrade.